browser/about_flags.cc

```diff
@@ -356,51 +356,58 @@
-          FEATURE_VALUE_TYPE(ai_chat::features::kPageContentRefine),         \
-      },                                                                     \
-      {                                                                      \
-          "brave-ai-chat-allow-private-ips",                                 \
-          "Private IP Addresses for Custom Model Endpoints",                 \
-          "Permits the use of private IP addresses as model endpoint URLs",  \
-          kOsWin | kOsMac | kOsLinux | kOsAndroid,                           \
-          FEATURE_VALUE_TYPE(ai_chat::features::kAllowPrivateIPs),           \
-      },                                                                     \
-      {                                                                      \
-          "brave-ai-chat-open-leo-from-brave-search",                        \
-          "Open Leo AI Chat from Brave Search",                              \
-          "Enables opening Leo AI Chat from Brave Search",                   \
-          kOsDesktop | kOsAndroid,                                           \
-          FEATURE_VALUE_TYPE(ai_chat::features::kOpenAIChatFromBraveSearch), \
+#define BRAVE_AI_CHAT_FEATURE_ENTRIES                                         \
+  EXPAND_FEATURE_ENTRIES(                                                     \
+      {                                                                       \
+          "brave-ai-chat",                                                    \
+          "Brave AI Chat",                                                    \
+          "Summarize articles and engage in conversation with AI",            \
+          kOsWin | kOsMac | kOsLinux | kOsAndroid,                            \
+          FEATURE_VALUE_TYPE(ai_chat::features::kAIChat),                     \
+      },                                                                      \
+      {                                                                       \
+          "brave-ai-chat-history",                                            \
+          "Brave AI Chat History",                                            \
+          "Enables AI Chat History persistence and management",               \
+          kOsWin | kOsMac | kOsLinux,                                         \
+          FEATURE_VALUE_TYPE(ai_chat::features::kAIChatHistory),              \
+      },                                                                      \
+      {                                                                       \
+          "brave-ai-host-specific-distillation",                              \
+          "Brave AI Host-Specific Distillation",                              \
+          "Enables support for host-specific distillation scripts",           \
+          kOsWin | kOsMac | kOsLinux,                                         \
+          FEATURE_VALUE_TYPE(ai_chat::features::kAIHostSpecificDistillation), \
```

Comment:
I would drop `AI` from this. There's an inconsistency here with `AI` vs `AIChat`, but this is already under `aichat` namespace so it's not really necessary here and most of the existing features do not have it.